index.bs

```diff
@@ -357,10 +357,12 @@ The presentation times of AV1 samples are given by the ISOBMFF structures. <asse
 
 <assert>The sample entry SHOULD contain a 'colr' box with a colour_type set to 'nclx'</assert>. <assert>If present, the values of colour_primaries, transfer_characteristics, and matrix_coefficients SHALL match the values given in the [=Sequence Header OBU=] (in the configOBUs field or in the associated samples) if the color_description_present_flag is set to 1</assert>. Similarly, <assert>the full_range_flag in the 'colr' box SHALL match the color_range flag in the [=Sequence Header OBU=]</assert>. <assert>When configOBUs does not contain a [=Sequence Header OBU=], this box with colour_type set to 'nclx' SHALL be present.</assert>
 
-<assert>For sample entries corresponding to HDR content, the MasteringDisplayColourVolumeBox 'mdcv' and ContentLightLevelBox 'clli' SHOULD be present</assert>, and <assert>their values SHALL match the values contained in the [=Metadata OBUs=] of type METADATA_TYPE_HDR_CLL and METADATA_TYPE_HDR_MDCV, if present (in the configOBUs or in the samples)</assert>.
+<assert>For sample entries corresponding to HDR content, the MasteringDisplayColourVolumeBox 'mdcv' and ContentLightLevelBox 'clli' SHOULD be present</assert>, and <assert>their values SHALL equal (except for different binary representations and precision issues, see the second note below) the values contained in the [=Metadata OBUs=] of type METADATA_TYPE_HDR_MDCV and METADATA_TYPE_HDR_CLL, if present (in the configOBUs or in the samples)</assert>.
```

Comment:
Cyril: Compared with https://github.com/AOMediaCodec/av1-isobmff/pull/141 (which you reviewed), this PR changed the following:

> equal (except for precision issues, see the second note below)

to:

> equal (except for different binary representations and precision issues, see the second note below)